### PR TITLE
Update customers with unit testing

### PR DESCRIPTION
**Pull request by ankitdas13**
command for runnning api from sdk folder : `bundle exec ruby test.rb`
command for runnning testing  : `bundle exec rake test`

for fetchTokens :  https://github.com/razorpay/razorpay-node/blob/master/documents/token.md#fetch-tokens-by-customer-id
Link : https://razorpay.com/docs/api/recurring-payments/upi/tokens/#22-fetch-tokens-by-customer-id

for fetchToken : https://github.com/razorpay/razorpay-node/blob/master/documents/token.md#fetch-particular-token
Link :  none

for deleteToken : https://github.com/razorpay/razorpay-node/blob/master/documents/token.md#delete-token
Link : https://razorpay.com/docs/api/recurring-payments/upi/tokens/#23-delete-tokens

### CODE REVIEW

File: test/razorpay/test_customer.rb [hunk 1]
```diff
@@ -41,5 +41,24 @@ def test_fetch_specific_customer
...
+      stub_get(%r{customers/#{@customer_id}/tokens$}, 'tokens_collection')
+      customers = Razorpay::Customer.fetch(@customer_id).fetchTokens
+      assert_instance_of Razorpay::Collection, customers, 'Customer not an instance of Razorpay::Customer class'
+      refute_empty customers.items, 'customers should be more than one'
+    end
+
+    def test_payments_fetch_token
+      stub_get(%r{customers/#{@customer_id}/tokens/#{@token_id}$}, 'fake_token')
+      customers = Razorpay::Customer.fetch(@customer_id).fetchToken(@token_id)
+      assert_equal @token_id, customers.id
```
Comment (arjunmenon) [inline test/razorpay/test_customer.rb:55]: nit: var name, token.id

File: lib/razorpay/customer.rb [hunk 2]
```diff
@@ -23,5 +23,17 @@ def self.edit(id, options = {})
...
       request.all options
     end
+
+    def fetchTokens
+      self.class.request.get "#{id}/tokens" 
+    end
+
+    def fetchToken(tokenId)
+      self.class.request.get "#{id}/tokens/#{tokenId}" 
+    end
```
Comment (arjunmenon) [inline lib/razorpay/customer.rb:33]: Can you link the relevant doc for this? In desc it is mentioned as https://razorpay.com/docs/api/recurring-payments/upi/tokens/#21-fetch-token-by-payment-id

File: documents/tokens.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+```
+-------------------------------------------------------------------------------------------------------
+
+### Fetch particular token
+```rb
+customerId = "cust_1Aa00000000004"
+
+tokenId = "token_Hxe0skTXLeg9pF"
+
+Razorpay::Customer.fetchToken(customerId, tokenId)
```
Comment (arjunmenon) [inline documents/tokens.md:134]: Pls rechk. Not same as implementation

File: documents/tokens.md [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+```
+
+-------------------------------------------------------------------------------------------------------
+
+### Fetch tokens by customer id
+
+```rb
+customerId = "cust_1Aa00000000004"
+
+Razorpay::Customer.fetchTokens(customerId)
```
Comment (arjunmenon) [inline documents/tokens.md:71]: Pls rechk. Not same as implementation

File: test/razorpay/test_customer.rb [hunk 5]
```diff
@@ -41,5 +41,24 @@ def test_fetch_specific_customer
       assert_instance_of Razorpay::Customer, customer
       assert_equal customer.id, @customer_id
     end
+
+    def test_payments_fetch_tokens
+      stub_get(%r{customers/#{@customer_id}/tokens$}, 'tokens_collection')
+      customers = Razorpay::Customer.fetch(@customer_id).fetchTokens
+      assert_instance_of Razorpay::Collection, customers, 'Customer not an instance of Razorpay::Customer class'
+      refute_empty customers.items, 'customers should be more than one'
```
Comment (arjunmenon) [inline test/razorpay/test_customer.rb:49]: nit: error msg, these are not customers.

**Review by arjunmenon — COMMENTED** [review]
Pls recheck example usage written for methods in documents

File: lib/razorpay/customer.rb (hunk 2 above)
Comment (ankitdas13) [reply lib/razorpay/customer.rb:36]: @arjunmenon i recently check again the above link and i got that this doc is not belongs to this function and there is no doc for this . i got this function from php sdk so i had replicated in other sdk (node, python)

File: test/razorpay/test_customer.rb (hunk 1 above)
Comment (ankitdas13) [reply test/razorpay/test_customer.rb]: done

File: documents/tokens.md (hunk 3 above)
Comment (ankitdas13) [reply documents/tokens.md]: it done now

File: documents/tokens.md (hunk 4 above)
Comment (ankitdas13) [reply documents/tokens.md]: it done now

File: test/razorpay/test_customer.rb (hunk 5 above)
Comment (ankitdas13) [reply test/razorpay/test_customer.rb]: changed that message and variable

File: lib/razorpay/customer.rb (hunk 2 above)
Comment (arjunmenon) [reply lib/razorpay/customer.rb:36]: <Internal> [Pls see](https://razorpay.slack.com/archives/C029ZL11PNK/p1648732059006459?thread_ts=1647927334.987909&cid=C029ZL11PNK) public docs are updated accordingly.

**Review by arjunmenon — APPROVED** [review]
LGTM. Pls see public docs are updated accordingly for APIs which are not there.

**Pull request merged**